requirements.txt

```diff
@@ -1,7 +1,7 @@
 branca
 folium
 numpy
-matplotlib
+matplotlib<3.3.3
```

Comment:
Great fix. Popups could also done with matplotlib -> svg -> popup. This way we could get rid of mpld3, but i thnk mpld figures  are fine. They also take less space compared to pure svg.

Reply:
I think this is ok solution until we do something better